lib/lazy_object.rb

```diff
@@ -27,4 +27,17 @@ def __target_object__
   def method_missing(method_name, *args, &block)
     __target_object__.send(method_name, *args, &block)
   end
+
+  def present?
+    !blank?
+  end
+
+  def blank?
+    __target_object__.respond_to?(:empty?) ? !!__target_object__.empty? : !__target_object__
+  end
+
+  # Forwards method calls to the target object.
```

Comment:
You can find unit tests here:
https://github.com/HornsAndHooves/lazy_object/blob/version_0_0_2/spec/lazy_object_spec.rb
BasicObject doesn't have any of this methods, so I think my changes are necessary.
http://ruby-doc.org/core-2.1.5/BasicObject.html